lib/internal/process.js

```diff
@@ -84,8 +84,7 @@ function setup_hrtime() {
         const needsBorrow = nsec < 0;
         return [needsBorrow ? sec - 1 : sec, needsBorrow ? nsec + 1e9 : nsec];
       }
-      throw new errors.TypeError('ERR_INVALID_ARG_TYPE',
-                                 'process.hrtime()', 'Array');
+      throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'time', 'Array');
```

Comment:
@BridgeAR I know how it can be rewritten, that is not the point (I am actually already running tests against a fix). This is more of a design decision whether we want a code `ERR_INVALID_ARRAY_LENGTH` or such.